yugabyte/charts
# Add PostgreSQL service

Also moved statefulset to separated file.
Testing plan:
- install clear chart: `helm upgrade --install --create-namespace -n sd-test yw .`
  > No service at this point
- enable PG service: `helm upgrade --install --create-namespace -n sd-test --set service.postgres=true yw .`
  > Service yw-postgres appeared
- check psql connection:
  - add sidecar pod
  - exec to it
  - install psql: `yum install -y postgresql`
  - try `psql -h yw-postgres` - prompt for password
- upgrade release with postgres disabled: `helm upgrade --install --create-namespace -n sd-test --set yugaware.service.postgres=false yw .`
  > No postgres service at this point
- delete release: `helm -n sd-test delete yw`
  > No orphaned objects at this point
